README.md

```diff
@@ -20,7 +20,7 @@ Building for Fedora
 
 To install build requirements, run following command:
 
-    dnf install check-devel cmake cppunit-devel gcc gcc-c++ glib2-devel gpgme-devel gtk-doc json-c-devel libmodulemd-devel librepo-devel libsolv-devel libsolv-tools make python2-devel python3-devel python2-nose python3-nose python2-sphinx python3-sphinx rpm-devel sqlite-devel swig libsmartcols-devel
+    dnf install check-devel cmake cppunit-devel gcc gcc-c++ glib2-devel gpgme-devel gtk-doc json-c-devel libmodulemd-devel librepo-devel libsolv-devel libsolv-tools make python2-devel python3-devel python2-nose python3-nose python2-sphinx python3-sphinx python3-breathe python3-breathe rpm-devel sqlite-devel swig libsmartcols-devel
```

Comment:
Why are we listing `python3-breathe` twice? Do we mean `python2-breathe python3-breathe`?